src/index.js

```diff
@@ -36,35 +31,31 @@ $('#js-names-button').click(function(e) {
   let player3 = new Player($('#js-input-player-3').val(), 3)
   game.players.push(player1, player2, player3)
   game.gameStart();
-  domUpdates.updatePlayerScore(player1, player2, player3)
   domUpdates.displayCluesIds(clues)
   domUpdates.displayCategories(clues)
   domUpdates.updatePlayerNames()
   $('#js-input-names').hide();
   $('#js-jeopardy-board').show();
-  // domUpdates.displayCurrentQuestion(e)
+  $('#js-players-heading').text(game.round.currentTurn.currentPlayer.name)
 
-    $('.card').on('click', function(e) {
-  
-      game.round.currentTurn.currentCard = $(e.target)[0].id;
+$('.card').on('click', function(e) {
+  game.round.currentTurn.currentCard = $(e.target)[0].id;
+  let question = $(e.target)[0].innerText;
+  let clue = clues.cards.find(clue => {
+  return clue.question === question;
+  })
 
+  game.round.currentTurn.currentClue = clue;
+})
 
-      let question = $(e.target)[0].innerText;
-      let clue = clues.cards.find(clue => {
-        return clue.question === question;
-      })
-      game.round.currentTurn.currentClue = clue;
-      console.log('clue object', clue)
-    })
-  $('#js-guess-button').click(function(e) {
-    e.preventDefault();
-    let guess = $('#js-guess-input').val();
-    let answer = game.round.currentTurn.evaluateGuess(guess)
-    console.log('answer', answer)
-    console.log(game.round.currentTurn.currentCard);
-    
-    domUpdates.giveFeedback(answer, game.round.currentTurn.currentCard);
-    game.round.changePlayer();
-  })
+$('#js-guess-button').click(function(e) {
```

Comment:
Good to see this adding of an event listener was extracted so multiple event listeners weren't added to the same element for each click.